network/src/main/java/uk/gov/android/network/client/KtorHttpClient.kt

```diff
@@ -165,6 +169,11 @@ class KtorHttpClient(
                         append(header.first, header.second)
                     }
                 }
+                url {
```

Comment:
@alex-bradbury I've rebased this on top of the changes, which has fixed the failing build.
Please can we get some unit tests for this functionality, then happy to get it merged 🙂